core_backend/tests/api/test.env

```diff
@@ -1,10 +1,6 @@
 POSTGRES_USER=postgres-test-user
 POSTGRES_PASSWORD=postgres-test-pw
 POSTGRES_DB=postgres-test-db
-POSTGRES_PORT=5433
```

Comment:
Why did you delete this? By default POSTGRES_PORT=5432 in app_config.py. However, in the make setup-test-db target we launch a container to be exposed to port 5433!

Reply:
Running `make tests` in core_backend is failing due to this line being removed.

Reply:
Will fix it